Yoast/wordpress-seo
# Added the loading SEMrush modal content

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Adds content for the SEMrush modal dialog for the scenario where the data coming from SEMrush needs to be loaded.

## Relevant technical choices:

* Introduce a simple `SemRushLoading` component in the `components/modals` directory

## Test instructions

This PR can be tested by following these steps:

- I have added the component to the RelatedKeyPhrasesModal for testing purposes. 
- To toggle the component, you can change the `isLoading` state to never be set to true while opening the modal, run `grunt build`, refresh and reopen the modal.
- Verify the content follows the design provided in the issue
